webpack.config.js

```diff
@@ -25,8 +25,13 @@ module.exports = {
         loaders: [
             {
                 test: /\.less$/,
-                loader: ExtractTextPlugin.extract("style-loader", "css-loader!less-loader")
-            }
+                loader: ExtractTextPlugin.extract('style-loader', 'css-loader!less-loader')
+            },
+            {
+                test: /\.tpl\.html$/,
+                loader: 'raw-loader',
+                exclude: /node_modules/
+            },
```

Comment:
You are the pro about webpack, but I am wondering , what is that doing now here ?